www/js/defaults.json

```diff
@@ -107,7 +107,8 @@
         "translationFont": "Muli",
         "gurbaniFont": "Gurbani Akhar Thick",
         "larivaarAssist": false,
-        "theme": false
+        "theme": false,
+        "greenScreen": false
```

Comment:
make sure to check "fresh installs" we usually run into issue when adding new defaults.